src/conventional/ionosonde_ascii2ioda.py

```diff
@@ -0,0 +1,415 @@
+#!/usr/bin/env python3
+
+import sys
+import os
+import time
+from datetime import datetime
+import numpy as np
+import logging
+
+# These modules need the path to lib-python modules
+import pyiodaconv.ioda_conv_engines as iconv
+from pyiodaconv.orddicts import DefaultOrderedDict
+
+os.environ["TZ"] = "UTC"
+
+# these are the unique values in the raw input file
+varDict = {'frequency': ['frequency', "float", 'm s-1'],                                         # chirp frequency output in MHz
+           'frequencyConfidence': ['frequencyConfidence', "float", 'fractional percent'],        # half-width of freq?
+           'height': ['height', "float", "m"],                                                   # range or radar chirp
+           'electronDensity': ['electronDensity', "float", 'number / m^3'],                      # number per volume
+           'electronDensityConfidence': ['electronDensityConfidence', "float", 'number / m^3'],  # uncertainty in number per volume
+           'ionosondeScalingParameter': ['ionosondeScalingParameter', "float", 'Unknown'],       # 3.71 - this is something about how the values are calculated
+           'ionosphericLayer': ['ionosphericLayer', "integer", 'IonosphericLayer']}              # F2   - this is the name of the layer
+
+# these are the MetaData common to each input
+locationKeyList = [("latitude", "float", "degrees_north"),
+                   ("longitude", "float", "degrees_east"),
+                   ("dateTime", "long", "seconds since 1970-01-01T00:00:00Z"),
+                   ("hmF2", "float", "Unknown"),             # height of F2 layer
+                   ("foF2", "float", "Unknown"),             # freq of F2 (in MHz)
+                   ("foF2Confidence", "float", "Unknown"),   # bandwidth?
+                   ("nmF2", "float", "Unknown"),             # number density at hmF2
+                   ("nmF2Confidence", "float", "Unknown"),
```

Comment:
"ionospheric_par_NmF2_uncertainty"   Uncertainty for the F2 peak electron density